code/modules/clothing/glasses/glasses.dm

```diff
@@ -155,4 +155,29 @@
 	name = "Optical Thermal Eyepatch"
 	desc = "An eyepatch with built-in thermal optics."
 	icon_state = "eyepatch"
-	item_state = "eyepatch"
\ No newline at end of file
+	item_state = "eyepatch"
+
+/obj/item/clothing/glasses/cold
+	name = "cold goggles"

```

Comment:
"golly i wonder why these goggles i got from the theatre autodrobe aren't protecting me from extreme heat and cold"

"maybe it's because it's a prop and i'm a fucking idiot"